rinex/src/navigation/health/gps.rs

```diff
@@ -1,86 +1,140 @@
-//! GPS Sv Health specifications
+//! GPS SV Health specifications
 use bitflags::bitflags;
 
-/// GPS Sv Health indication
+/// GPS SV Health indication
 #[derive(Debug, Clone, PartialEq, PartialOrd)]
 #[cfg_attr(feature = "serde", derive(Serialize))]
 pub enum GPSHealth {
-    /// GPS Legacy Sv Health indication
-    LNAV(LegacyHealth),
-    /// GPS CNAV Sv Health indication
-    CNAV(CivilianHealth),
-    /// GPS CNV2 Sv Health indication
-    CNV2(Civilian2Health),
+    /// GPS Legacy SV Health indication
+    LNAV(LNAVHealth),
+    /// GPS CNAV SV Health indication
+    CNAV(CNAVHealth),
+    /// GPS CNV2 SV Health indication
+    CNV2(CNAVHealth),
 }
 
 impl Default for GPSHealth {
     fn default() -> Self {
-        Self::LNAV(LegacyHealth::default())
+        Self::LNAV(LNAVHealth::default())
     }
 }
 
 impl GPSHealth {
-    /// Unwraps self as [`LegacyHealth`] indicator
-    pub(crate) fn lnav(&self) -> Option<&LegacyHealth> {
+    /// Unwraps self as [`LNAVHealth`] indicator
+    pub(crate) fn lnav(&self) -> Option<&LNAVHealth> {
         match self {
             Self::LNAV(h) => Some(h),
             _ => None,
         }
     }
-    /// Unwraps self as [`CivilianHealth`] indicator
-    pub(crate) fn cnav(&self) -> Option<&CivilianHealth> {
+    /// Unwraps self as [`CNAVHealth`] indicator
+    pub(crate) fn cnav(&self) -> Option<&CNAVHealth> {
         match self {
             Self::CNAV(h) => Some(h),
             _ => None,
         }
     }
-    /// Unwraps self as [`Civilian2Health`] indicator
-    pub(crate) fn cnv2(&self) -> Option<&Civilian2Health> {
+    /// Unwraps self as [`CNAVHealth`] indicator
+    pub(crate) fn cnv2(&self) -> Option<&CNAVHealth> {
         match self {
             Self::CNV2(h) => Some(h),
             _ => None,
         }
     }
 }
 
-bitflags! {
-    /// GPS Legacy Sv Health indication.
-    /// See [Bibliography::RINEX3] for more information.
-    #[derive(Default, Debug, Clone, PartialEq, PartialOrd)]
-    #[cfg_attr(feature = "serde", derive(Serialize))]
-    pub struct LegacyHealth: u64 {
-        const L1Healthy = 0x01;
-        const L2Healthy = 0x02;
-        const L5Healthy = 0x04;
-    }
+/// GPS Legacy SV Health indication.
```

Comment:
Similar to the Galileo struct, I also made this a struct as the lower 5 bits are actually a 5 bit integer

To tell health of just L1 or L2 some complex switch would be needed